yarn.lock

```diff
@@ -367,15 +367,20 @@
     minimatch "^3.1.2"
     strip-json-comments "^3.1.1"
 
-"@ethereumjs/common@^2.4.0", "@ethereumjs/common@^2.6.4", "@ethereumjs/common@^2.6.5":
+"@ethereumjs/common@^2.4.0", "@ethereumjs/common@^2.6.4":
```

Comment:
I am wondering why this is still included (the same for `ethereumjs/tx` below). Is it because `@ethereumjs/rlp` is still used?

Reply:
I think I checked that before and it's actually ganache and maybe harhat that pull those in^^